pyproject.toml

```diff
@@ -4,6 +4,8 @@ build-backend = "setuptools.build_meta"
 
 [tool.ruff]
 
+src = ["pasqal-cloud", "pulser-pasqal"]
```

Comment:
I am not here it is needed as it i scanning the main directory, didn't it work properly ?

Reply:
Without this the imports in the `tests` see `pasqal-cloud` and `pulser-pasqal` as third-party libraries and enforce an import order that, to me, doesn't make much sense.

Reply:
I see, it makes sense indeed